pkg/api/response/response.go

```diff
@@ -239,6 +239,22 @@ func Err(err error) *NormalResponse {
 	return resp
 }
 
+// ErrWithFallback uses the information in an errutil.Error if available
+// and otherwise falls back to the status and message provided as
+// arguments.
+//
+// The signature is equivalent to that of Error which allows us to
+// rename this to Error when we're confident that that would be safe to
+// do.
+func ErrWithFallback(status int, message string, err error) *NormalResponse {
```

Comment:
```suggestion
func ErrOrFallback(err error, status int, message string) *NormalResponse {
```

Dual nitpicking here: 
* `Or` instead of `With`, since the function doesn't return both;
* the parameter order should match the function name (error then the fall back metadata)

Reply:
Or seems reasonable, I’ll change that. As for the order, the comment explains why it’s the particular order: otherwise it’s deviating from the regular Error method that I really want to match to make mass refactoring easier. I agree that it’s not ideal, but I have a sense that that alignment will help adoption.